scripts/start_cluster.sh

```diff
@@ -26,6 +26,8 @@ if [[ "$OSTYPE" == "linux-gnu"* ]]; then
   fi
 fi
 
+export LD_LIBRARY_PATH=$PWD/internal/core/output/lib:$LD_LIBRARY_PATH
```

Comment:
Why we need this? Btw, $PWD seems not always correct if we don't execute this script in project's root directory.